### PR TITLE
KEYCLOAK-15291 Revise Server Installation guide to meet Red Hat standards

### PR DESCRIPTION
**Pull request by andymunro**
@hmlnarik, @pdrozd Can someone review this and approve.

### CODE REVIEW

**Comment by andymunro** [comment]
@pdrozd, @pskopek Please prioritize this PR for 7.5.1.

**Review by pedroigor — DISMISSED** [review]

**Comment by stianst** [comment]
@andymunro @pdrozd @pskopek we do NOT backport refactoring/enhancements to 7.5, these changes will come in 7.6.

File: server_installation/topics/cache/clear.adoc [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
-On this page you can clear the realm cache, the user cache or cache of external public keys.
+You can clear the realm cache, user cache, or the external public keys.
+
+.Procedure
+
+. Log into the Admin Console.
+
+. Click *Realm Settings*.
+
+. Click the *Cache Config* tab.
```
Comment (ikhomyn) [inline server_installation/topics/cache/clear.adoc]: @andymunro Looks good. Just one thing, can we change here tab name as per screenshot? Thanks
```suggestion
. Click the *Cache* tab.
```
Screenshot from 2021-11-02 15-20-34
s

File: server_installation/topics/operating-mode/standalone-ha.adoc [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-=== Standalone Clustered Mode
+=== Using standalone clustered mode
 
-Standalone clustered operation mode is for when you want to run {project_name} within a cluster.  This mode
-requires that you have a copy of the {project_name} distribution on each machine you want to run a server instance.
+Standalone clustered operation mode applies when you want to run {project_name} within a cluster.  This mode
+requires that you have a copy of the {project_name} distribution on each machine where you want to run a server instance.
 This mode can be very easy to deploy initially, but can become quite cumbersome. To make a configuration change
-you'll have to modify each distribution on each machine.  For a large cluster this can become time consuming and error prone.
+youhave to modify each distribution on each machine.  For a large cluster, this mode can become time consuming and error prone.
```
Comment (ikhomyn) [inline server_installation/topics/operating-mode/standalone-ha.adoc]:
```suggestion
you have to modify each distribution on each machine.  For a large cluster, this mode can become time consuming and error prone.
```

**Review by ikhomyn — COMMENTED** [review]
@andymunro Looks good, well done. I have found only some minor typos.

**Comment by andymunro** [comment]
Thanks, @ikhomyn for the corrections.  I have made those updates.

**Comment by andymunro** [comment]
@ikhomyn  Please let me know when you approve this or if you request more changes.

**Review by stianst — APPROVED** [review]

**Pull request merged**